internal/librarian/release_init.go

```diff
@@ -113,10 +116,23 @@ func (r *initRunner) run(ctx context.Context) error {
 		commitMessage: "",
 		prType:        release,
 	}
-	if err := commitAndPush(ctx, commitInfo); err != nil {
+	if prMetadata, err = commitAndPush(ctx, commitInfo); err != nil {
 		return fmt.Errorf("failed to commit and push: %w", err)
 	}
 
+	gitHubRepo, err := github.FetchGitHubRepoFromRemote(r.repo)
+	if err != nil {
+		return fmt.Errorf("unable to add retrieve the github repo to add `release:pending` label: %w", err)
+	}
+
+	if commitInfo.cfg.Push {
```

Comment:
Is there a way to put this part into `commitAndPush`?

I think adding labels should stay together with creating pull request.